src/control/server/mgmt_svc.go

```diff
@@ -530,9 +530,13 @@ func (svc *mgmtSvc) PoolDeleteACL(ctx context.Context, req *mgmtpb.DeleteACLReq)
 func (svc *mgmtSvc) BioHealthQuery(ctx context.Context, req *mgmtpb.BioHealthReq) (*mgmtpb.BioHealthResp, error) {
 	svc.log.Debugf("MgmtSvc.BioHealthQuery dispatch, req:%+v\n", *req)
 
-	mi, err := svc.harness.GetMSLeaderInstance()
-	if err != nil {
-		return nil, err
+	var mi *IOServerInstance
+	for _, i := range svc.harness.Instances() {
+		mi = i
+	}
```

Comment:
Thinking about this a bit more, I don't think we can take the naive approach here, unfortunately. The request is for a specific device UUID, right? If we pick the wrong instance, then this request will fail because the instance hasn't bound that device.